Improvements as per discussions and test plan notes (#46)
AMWA-TV/ms-05-02: docs/Framework.md

### NcObject

> NcObject is the base abstract control class for any control class in the control model.  

Reviewer: Possible simplification/clarification
```suggestion
NcObject is the abstract base class for all classes in the control model.  
```

> Further explanations and normative references are provided in a dedicated [NcObject](NcObject.md) section.

Reviewer: Suggested simplification
```suggestion
Further explanations and normative references are provided in the [NcObject](NcObject.md) section.
```

### Workers

> Further explanations and normative references are provided in a dedicated [Workers](Workers.md) section.

Reviewer: Suggested simplification
```suggestion
Further explanations and normative references are provided in the [Workers](Workers.md) section.
```

#### NcWorker

> NcWorker is the base worker control class for any worker control class in the control model.

Reviewer: Suggested simplification
```suggestion
NcWorker is the base class for any worker control class in the control model.
```
